src/NzbDrone.Core/Download/Clients/DownloadStation/TorrentDownloadStation.cs

```diff
@@ -272,7 +272,14 @@ protected ValidationFailure TestOutputPath()
         {
             try
             {
-                var downloadDir = GetDownloadDirectory();
+                var downloadDir = GetDefaultDir();
+
+                if (downloadDir == null)
+                {
+                    return new NzbDroneValidationFailure(string.Empty, "No default destination configured. You must manually set it up into download station settings.");
```

Comment:
Yes, coz filling in a directory there would _also_ fix the issue for the user. So it's the most sensible spot. imho.